src/App.tsx

```diff
@@ -1,15 +1,27 @@
 import Clock from './components/Clock/Clock';
 import styles from './App.module.scss';
+import targetTime from './utils';
 
 function App() {
+  const getClientUrl = () => {
+    const urlCurrent = window.location.href;
+    const clientTtimeStamp = new Date().getTime();
+    const clientTimeZone = Intl.DateTimeFormat().resolvedOptions().timeZone;
+    const urlClient = `${urlCurrent}/?ts=${clientTtimeStamp}&tz=${clientTimeZone}`;
+    return urlClient;
+  };
```

Comment:
If this code is for testing, you can remove this code now

Reply:
I assumed that this code would be needed to determine the user's location.

Reply:
I get `clientTimeZone` - that is indeed gives us current time zone, but other things don't make much sense. No need in building an URL for instance

Reply:
Well understood